build.gradle

```diff
@@ -60,6 +60,8 @@ dependencies {
 	modRuntimeOnly('me.djtheredstoner:DevAuth-fabric:1.1.0') {
 		exclude group: 'net.fabricmc', module: 'fabric-loader'
 	}
+
+	include api('net.fabricmc:mapping-io:0.5.1')
```

Comment:
I'm worried that this may clash with a version of mapping-io that is used inside Fabric loader. Does Fabric loader also JiJ it or does it just include it in the jar? If it's JiJ it should be fine as loader will pick the latest version, otherwise I'm worried as we don't want to break loader if it requires a later version of mapping-io. I'm also worried because mapping-io is still in a 0.* phase, which means 0.6 could contain breaking changes, which could break our mod on future Fabric loader versions, which is a problem because new loader versions can be used on old mc versions. To solve this, we could repackage mapping-io.

Reply:
Loader shadows, relocates, and minimizes MIO.